homeassistant/components/asuswrt/__init__.py

```diff
@@ -125,6 +125,14 @@ async def async_setup_entry(hass: HomeAssistantType, entry: ConfigEntry):
 
     router.async_on_close(entry.add_update_listener(update_listener))
 
+    # Device ID was changed. Link up old device to new one.
+    dev_reg = device_registry.async_get(hass)
+    legacy_entry = dev_reg.async_get_device((DOMAIN, "AsusWRT"))
+    if legacy_entry and len(legacy_entry.identifiers) == 1:
+        dev_reg.async_update_device(
+            legacy_entry.id, new_identifiers=(DOMAIN, entry.entry_id)
```

Comment:
We only allow a single config entry. Is this still needed?

Reply:
There should be no reason to allow a single config entry :)

Reply:
At this moment it is not possible to have a valid unique ID to manage multiple config entry

Reply:
Right, that's why we use the entry ID.

Reply:
We should change this limit then, otherwise there's no point to this change here.

Reply:
The only way to do this is having a method in the Asus library that provide a unique ID for the router, as I know it does not exist.

Reply:
@ollo69 that's why we use the config entry ID as a placeholder unique ID. You should still be able to connect to 2 routers, if one is on another network.